### PR TITLE
Updating Python scripts

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Python main scripts have been updated to make them as similar as possible between applications. A unique python script is used for both serial and parallel execution.

### CODE REVIEW

File: kratos.gid/apps/Structural/python/KratosStructural.py [hunk 1]
```diff
@@ -1,110 +1,88 @@
...
-        used_time = time_fp - time_ip
-        print("::[KSM Simulation]:: [ %.2f" % round(used_time,2),"s", process," ] ")
-
-#### TIME MONITORING END ####
-
-#### PARSING THE PARAMETERS ####
-
-# Import define_output
+from KratosMultiphysics import *
+from KratosMultiphysics.ExternalSolversApplication  import *
```
Comment (loumalouomega) [inline kratos.gid/apps/Structural/python/KratosStructural.py]: Why import whit *?

Comment (rubenzorrilla) [inline kratos.gid/apps/Structural/python/KratosStructural.py]: A time ago we decided to do the import with * in the main scripts.

Comment (RiccardoRossi) [inline kratos.gid/apps/Structural/python/KratosStructural.py]: i think that the argument was "simplicity". However in the solvers it should be used without the *

File: kratos.gid/apps/EmbeddedFluid/python/KratosFluid.py [hunk 2]
```diff
@@ -76,11 +80,12 @@
...
-if(verbosity > 1):
-    print("")
-    print(main_model_part)
-    for properties in main_model_part.Properties:
-        print(properties)
+if(echo_level > 1):
+    if (parallel_type == "OpenMP") or (mpi.rank == 0):
+        print("")
+        print(main_model_part)
+        for properties in main_model_part.Properties:
```
Comment (RiccardoRossi) [inline kratos.gid/apps/EmbeddedFluid/python/KratosFluid.py]: why do we want to print this? is it for debug purposes?

Comment (rubenzorrilla) [inline kratos.gid/apps/EmbeddedFluid/python/KratosFluid.py]: Probably we can remove this one, or maybe print it when echo_level is larger than one (as is done with other print statements).

File: kratos.gid/apps/Structural/python/KratosStructural.py (hunk 1 above)
Comment (loumalouomega) [inline kratos.gid/apps/Structural/python/KratosStructural.py]: OK

**Comment by jginternational** [comment]
Will this work with the [Release](https://github.com/KratosMultiphysics/Kratos/tree/Release-5.1) branch?

**Comment by rubenzorrilla** [comment]
Indeed, this must be in the release branch.

**Pull request merged**